src/std/os/kqueue.ss

```diff
@@ -100,6 +102,16 @@ package: std/os
   kevent_udata kevent_udata_set
   ev_set)
 
+(cond-expand
+  ((not darwin)
+    (extern EVILT_DEVICE NOTE_TRUNCATE NOTE_EOF NOTE_CHANGE)))
+
+(cond-expand
```

Comment:
can you merge these two cond-expands?
````
(cond-expand
 (darwin ...)
 (else ...))
```

Reply:
I kept them separate, as they have a different meaning. The first one with `(not darwin)` elides some identifiers from the collection of `extern` identifiers. The second `cond-expand` with the positive `darwin` clause transfers cond expand features from gerbil to Gambit and conceptually belongs to the following `begin-foreign` block.

But I will merge them if you prefer.

Reply:
fair enough, that's good justification :) -- no need to merge.